### PR TITLE
Best practices

### PR DESCRIPTION
**Pull request by NATASHA-ct**

### CODE REVIEW

File: mods/addremove.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    const todo = todos.find((todo) => todo.index === index);
+    todo.description = e.target.outerText;
+    localStorage.setItem('todos', JSON.stringify(todos));
+  });
+  remove.addEventListener('click', (e) => {
+    e.preventDefault();
+    const newTodos = removeTodos((todo) => todo.index !== index, todos);
+    localStorage.setItem('todos', JSON.stringify(newTodos));
+    setToDos(newTodos);
+    // eslint-disable-next-line no-use-before-define
```
Comment (aneallaryea100) [inline mods/addremove.js:70]: - [ ] Can you kindly remove commented codes and also in using commented codes, to make it more descriptive.

**Review by aneallaryea100 — APPROVED** [review]
**Great job so far**

File: mods/addremove.js (hunk 1 above)
Comment (NATASHA-ct) [reply mods/addremove.js:70]: Thank you for your review

**Comment by NATASHA-ct** [comment]
### Thank you for your review @aneallaryea100 
I managed to make the change you requested:

- Added comments to make my code more readable

Can you kindly make another review for me.

**Review by aneallaryea100 — APPROVED** [review]
Hello @NATASHA-ct 
Great work 🥇 🥇 ✅ ✅ 
You can merge your branch now.

All the best. 👋

**Pull request merged**